Sources/protoc-gen-swift/EnumCaseGenerator.swift

```diff
@@ -0,0 +1,56 @@
+// Sources/protoc-gen-swift/EnumGenerator.swift - Enum case logic
```

Comment:
And this is why it's not a great idea to include the name/path of a source file in the file...